File: kitchen-web/src/app/model/book.ts
export class Book {
    id: number;
    name: string;
    description: string;
    iconUrl: string;
    bookListIcon: string;
    bookLongDescription: string;
    category: string;
    chapterCount: number;
    releasedAt: Date;
    createdBy: string;
    creationDate: Date;
    constructor() {
        this.name = '';
        this.description = '';
        this.iconUrl = '';
        this.bookListIcon = '';
        this.bookLongDescription = '';
        this.category = '';
        this.chapterCount = 0;
        this.releasedAt = new Date();
        this.createdBy = 'FrontEnd';
        this.creationDate = new Date();
    }
}

File: kitchen-api/Models/Book/Book.cs
using System;
namespace kitchen_api.Models
{
    public class Book
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public string IconUrl { get; set; }
        public string BookListIcon { get; set; }
        public string BookLongDescription { get; set; }
        public string Category { get; set; }
        public int ChapterCount { get; set; }
        public DateTime ReleasedAt { get; set; }
        public string CreatedBy { get; set; }
        public DateTime CreationDate { get; set; }
    }
}